ta/arch/arm/ta.ld.S

```diff
@@ -32,12 +32,32 @@ SECTIONS {
 		*(.gnu.linkonce.t.*)
 		/* Workaround for an erratum in ARM's VFP11 coprocessor */
 		*(.vfp11_veneer)
+#ifdef ARM32
+		PROVIDE(__gnu_mcount_nc = __gnu_mcount_nc__utee);
```

Comment:
Wouldn't it work to use a define instead of `__gnu_mcount_nc`/`_mcount` to get rid of the ifdeffing here and below?